app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,48 @@
+class HackdaysController < ApplicationController
+	
+	def create
+		@hackday = Hackday.new(hackday_params)
+		if @hackday.save
+			flash[:success] = "successfully added a hackday"
+			redirect_to @hackday
+		else
+			flash[:danger] = "failed to add a hackday"
+			redirect_to root_url
```

Comment:
if you redirect here then the data the user entered will disappear. which means they cant just edit and fix it.